### PR TITLE
Add support for symfony/process < v4

### PR DESCRIPTION
**Pull request by ngugijames**
See #8

### CODE REVIEW

File: src/GitVersionHelper.php [hunk 1]
```diff
@@ -40,7 +40,13 @@ public static function getVersion()
         $fail = false;
         if (class_exists('\Symfony\Component\Process\Process')) {
             try {
-                $process = Process::fromShellCommandline($command, $path);
+
```
Comment (tremby) [inline src/GitVersionHelper.php]: Just remove this new empty line and squash your commits, please.

**Review by tremby — CHANGES_REQUESTED** [review]
Looks good, thank you.

I haven't tested this; haven't used Laravel in quite some time. Have you verified it works with 5.5 and with later versions too?

Just one nitpick.

**Comment by tremby** [comment]
I notice that I (or a contributor) did add a version 4+ requirement for symfony/process in the composer.json suggestions section. Please update that accordingly as part of this changeset. https://github.com/tremby/laravel-git-version/blob/master/composer.json#L8

File: src/GitVersionHelper.php (hunk 1 above)
Comment (ngugijames) [reply src/GitVersionHelper.php]: Is it okay now?

**Review by tremby — APPROVED** [review]

**Comment by tremby** [comment]
I've [squashed your commits](https://github.com/tremby/laravel-git-version/commit/52e423040f3615d5ccd21a45afcbaf4f52661be2) and merged. I tagged [v1.2.2](https://github.com/tremby/laravel-git-version/tree/v1.2.2).

Thanks!

**Pull request closed without merging**